### DISCUSSION TITLE
What does the `_rsc` query really do? And where are the static routes cached?

### DISCUSSION THREAD

**Discussion by kavitang** [vercel/next.js#52524] [Help]
### Summary

Ever since the upgrade that added `_rsc` query to our requests, we have been experiencing a slow down on some pages that load a lot of data. Some of the routes are actually static routes, but these still fire a `_rsc` query, which then seems to be loading the entire route, which again is slow because it loads a lot of data. So this then ends up causing our pages to load very slowly sometimes, as once the cache is set it no longer is an issue. But every cold start of the serverless function for this route results in this cache being reset.

In the screen shot attached, we are building this route statically during build with `generateStaticParams` but for some reason this makes no difference as the first request this triggers the full execution of the route.. 

<img width="1370" alt="Screenshot 2023-07-10 at 4 50 06 PM" src="https://github.com/vercel/next.js/assets/96101675/a6e7acdc-a7b6-4889-8c01-512a93f8a74e">

I could not find too much information in the docs about the use of `_rsc` and what it does exactly. Any insights into what I'm missing here would be very helpful.  Any links I can go through etc are also very welcome, thanks!

PS: I know that we should look at reducing the amount of data being loaded to improve performance, and we are working on that, but I would like to get better understanding of what is happening here. Why are the statically built routes not just returning cached content right away? Why is the `/gallery/[[...space]].rsc` function executing for this route at all?

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:58 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6020
    Binaries:
      Node: 18.14.2
      npm: 9.5.0
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 13.4.7
      eslint-config-next: 13.4.7
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.3
```

**Comment by solvicode** [comment]
Same issue happening for me. I think this github issue summarises things quite succinctly.

I am quite surprised that this kind of functionality is allowed to occur...

**Reply by luckyship** [reply]
same issue for me, i start the nextjs project on localhost, i want to know why this request cost so long time?
![image](https://github.com/vercel/next.js/assets/22162049/3b1099c2-93e9-4c3a-9b14-2538e52ca5df)

**Comment by RicSala** [comment]
Same issue, but nobody seems to care...
